vrecord

```diff
@@ -1184,67 +1194,74 @@ else
     middleoptions+=(-color_range mpeg)
 fi
 
-if [[ "${qctoolsxml_choice}" = "Yes" ]] ; then
-    if [[ ! "$(which qcli)" ]] ; then
```

Comment:
This test was moved further down, but I suggest to keep it before the digitization commands. Otherwise the digitization commands will simply fail and provide a much messier output. Not blocking, but if unresolved here I'd suggest to resolve soon.